daemon/inertiad/crypto/totp.go

```diff
@@ -14,12 +14,13 @@ const (
 	totpSecretSize       = 10
 	totpDigits           = 6
 	totpAlgorithm        = otp.AlgorithmSHA1
-	totpNoBackupCodes    = 10
+	// TotpNoBackupCodes is the number of backup codes per user
+	TotpNoBackupCodes    = 10
```

Comment:
looks like gofmt isnt liking the spacing here: https://travis-ci.org/ubclaunchpad/inertia/jobs/449082500#L524

Reply:
once this is fixed ill merge this 👍 thanks for update!

Reply:
woops, should be fixed